amethyst_assets/src/prefab/system.rs

```diff
@@ -129,8 +129,8 @@ where
 
                         children
                             .entry(parent)
-                            .or_insert(vec![])
-                            .push(new_entity.clone());
+                            .or_insert_with(|| vec![])
```

Comment:
This should probably be `Vec::new` instead.

Reply:
```suggestion
                            .or_insert_with(Vec::new)
```

Reply:
Not sure what the difference is so I left it as is. Can I replace all instances of `vec![]` with `Vec::new()`.

Reply:
Ah, what I meant is not a closure; you can pass the function directly and it will coerce into a closure (I think clippy will even tell you that).

Reply:
Ah I see what you mean. It's fixed now.